### PR TITLE
Coordinate Data of all Stations

### PR DESCRIPTION
**Pull request by traten**
- [x] returns coordinate data as JSON of all stations
- [ ] Optimize time
This works, but is _incredibly_ slow, at least for me. Still trying to figure out a way to make it quicker.

### CODE REVIEW

File: api/app.py [hunk 1]
```diff
@@ -30,3 +32,29 @@ def coords_test():
...
+    units = 'english'
+    time_zone = 'gmt'
+    application = 'uf_tides'
+    res_format = 'json'
+
+    stationIds = open("Database/station_ids.txt", "r")
+    resDictList = []
+    
+    for x in stationIds:
+        print(x)
```
Comment (zmjackson) [inline api/app.py]: We could probably remove this print statement, yes?

File: api/app.py [hunk 2]
```diff
@@ -30,3 +32,29 @@ def coords_test():
...
+    datum = 'STND'
+    units = 'english'
+    time_zone = 'gmt'
+    application = 'uf_tides'
+    res_format = 'json'
+
+    stationIds = open("Database/station_ids.txt", "r")
+    resDictList = []
+    
+    for x in stationIds:
```
Comment (zmjackson) [inline api/app.py]: I think it would be cleaner to do ```for station in stationIds``` here, since you just reassign it later

File: api/app.py [hunk 3]
```diff
@@ -30,3 +32,29 @@ def coords_test():
...
+        print(x)
+        station = x
+        req = server + '?' + '&'.join(['date=' + date, 'station=' + station, 'product=' + product, 'datum=' + datum,
```
Comment (zmjackson) [inline api/app.py:52]: Probably no need to do a join here, I would just use the request string and format in the station variable.

**Review by zmjackson — COMMENTED** [review]
Looks great, thanks for doing this! I left a few comments. One general thing is that Python code typically uses snake_case rather than camelCase for functions and variables. Soon I'm going to add a CI step that checks formatting stuff like that, but for now I think we should probably just try to keep it consistent the best we can.

File: api/app.py (hunk 3 above)
Comment (traten) [reply api/app.py:52]: I'm a little confused what you mean by this one, are you suggesting doing something like this:
        `req = server + '?' + 'station=' + station`
where only the server is apart of the request message?

Comment (zmjackson) [reply api/app.py:52]: It's all good, it works for now! I'll merge it.

Comment (traten) [reply api/app.py:52]: Sounds good, I tried to do what i typed above and it didn't seem to work, not sure if the api request requires the whole string (date, product, etc) to return data

**Pull request merged**